src/log.c

```diff
@@ -266,6 +266,30 @@ void info_int( const char *file, const char *function, const unsigned long line,
 // ----------------------------------------------------------------------------
 // error log
 
+#ifdef WITHOUT_C11_GENERIC
```

Comment:
@alonbl this is where it is used.

What I need is a solution for `strerror_r`. The glibc version returns `char *`, while all others return `int`. The problem is that the glibc version might not use the `buf` supplied at all. So, we really need to know which version the current system has.